app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
I agree. We need to re-create the derivatives for those that have a mime type like "application/octet-stream" for jpeg. This should be the right only way to fix the issue since we'll never know how other institutes like Calisphere utilize our jpeg images and the Jhove metadata on solr or extract the Jhove metadata from the image files themselves. Those files are damage in some extension though it could be open, otherwise we can't explain why they have the wrong mime type and format.